core/gulpfile.js

```diff
@@ -1,217 +0,0 @@
-"use strict";
-
-require('es6-shim');
-var spawn = require('child_process').spawn;
-
-var fs = require("fs");
-var join = require('path').join;
-var gulp = require('gulp');
-var server = require('gulp-express');
-// var livereload = require('gulp-livereload');
-// var plumber = require('gulp-plumber');
-var browserify = require('browserify');
-var source = require("vinyl-source-stream");
-// var watchify = require('watchify');
-var generateSqlDefinition = require('sql-generate');
-
-
-var conString = "postgres://postgres:" + process.env.POSTGRES_PASSWORD + "@" + process.env.DB_PORT_5432_TCP_ADDR + ":5432/postgres";
-var sqlOptions = {
-    dsn: conString,
-    omitComments: true
-};
-
-var generateDefinitions = function() {
-    return new Promise(function(resolve, reject) {
-        generateSqlDefinition(sqlOptions, function(err, stats) {
-            if (err) {
-                console.error(err);
-                reject(err);
-            }
-            fs.writeFileSync("./database/management/declarations.js", stats.buffer);
-            resolve();
-        });
-    });
-}
-
-gulp.task('init', function () {                                 
-
-    // wait database to be created 
-    var interval = setInterval(function(){
```

Comment:
setInterval was not a good idea because `databaseClientP` may resolve after 10 intervals (resulting in drop/create being called 10 times in non-deterministic order, leading to an unclear state for the database)